python/ray/train/v2/_internal/util.py

```diff
@@ -16,7 +16,6 @@
 
 import ray
 from ray.train._internal.utils import count_required_parameters
-from ray.train.v2._internal.execution.callback import Callback
```

Comment:
I refactored the `invoke_callbacks_context_managers` a bit to fix a cyclical dependency issue.